### DISCUSSION TITLE
user-guide/backup

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#81] [文档评论区]
# user-guide/backup

我们强烈建议您定期备份 Zotero 数据目录。虽然同步是确保您可以在计算机发生问题时恢复库的好方法，但它并不能完全替代正确的备份：Zotero 服务器仅存储库的最新版本，并且只需要一个（可能是自动）同步以更改服务器副本（尽管可以从 Zotero 的自动备份恢复一些无意的更改）。 在本页中，我们首先介绍 Zotero 运行所需的三类文件，然后介绍若干种...

https://zotero-chinese.com/user-guide/backup.html

**Comment by Cooper1023** [comment]
蹲一个插件备份

**Reply by northword** [reply]
Chikit-L 更新辽

**Comment by llllhd** [comment]
> 该目录还包含一个 storage 文件夹，其中包含 8 个字符的子文件夹（例如N7SMB24A），其中包含所有文件附件，例如 PDF、网页快照、音频文件或您导入的任何其他文件。 （链接的文件不会复制到此子文件夹中。）

storage下的子文件夹可以不使用默认的随机命名，自定义命名规则吗？

**Reply by northword** [reply]
子文件夹不能修改命名规则

**Comment by ahhuipt** [comment]
请教一下，如果是zotero6升级到zotero7用Tara插件会出错吗？

**Comment by Bit0r** [comment]
```shell
tar -cavf zotero.tzst -C ~ .zotero Zotero
```

**Reply by Bit0r** [reply]
恢复用
```shell
tar -xvf zotero.tzst -C ~
```
